### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Faris90** [Malvads/AgarRepacker#1]
[!] https://github.com/Malvads/
[!] This tool has been developed by Malvads (TheGexi)
[!] Proxy Server Port: 3321
[!] Proxy Server Pointing To: wss://live-arena-qhl9j3.agar.io:443
[!] You are now ready :) enjoy!
D:\Downloads\AgarRepacker-master (2)\AgarRepacker-master\server.js:25
        this.version = res.match(/var\sversionString="(\d.\d.\d)";/)[1]
                                                                    ^

TypeError: Cannot read properties of null (reading '1')
    at AgarIO.getVersion (D:\Downloads\AgarRepacker-master (2)\AgarRepacker-master\server.js:25:69)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

**Comment by malvads** [comment]
Hi! This project is currently not supported. It was created a long time ago, back when the protocol version was 18. If you want this to work again, you'll need to reverse engineer the protocol once more to see if there have been any changes and make the corresponding adaptations. Sorry for any inconvenience.

**Comment by Faris90** [comment]
@malvads well it seems agar.io pc has slowed down development

**Comment by malvads** [comment]
yeah, agar.io is pretty much dead, it was fun to work on this haha @Faris90
